app/src/main/java/info/nightscout/androidaps/plugins/general/overview/OverviewFragment.kt

```diff
@@ -832,6 +833,8 @@ class OverviewFragment : DaggerFragment(), View.OnClickListener, OnLongClickList
         binding.activeProfile.text = profileFunction.getProfileNameWithRemainingTime()
         binding.activeProfile.setBackgroundColor(profileBackgroundColor)
         binding.activeProfile.setTextColor(profileTextColor)
+        val drawableLeft: Array<Drawable?> = binding.activeProfile.compoundDrawables
+        if (drawableLeft[0] != null) profileTextColor.let { drawableLeft[0]!!.setTint(it) }
```

Comment:
drawableLeft[0]?.let { it.setTint(profileTextColor) }

Reply:
the same bellow

Reply:
Oh yes thx, mine was a little bit to complicated

Reply:
simlified statement and former code